slu/slu/src/controller/prediction.py

```diff
@@ -32,8 +32,9 @@ def update_input(w: XLMRWorkflow, value: str) -> None:
 )
 
 
-def update_entities(workflow: XLMRWorkflow, entities: List[BaseEntity]):
-    workflow.output[const.ENTITIES] = entities
+def update_entities(workflow: XLMRWorkflow, extra_entities: List[BaseEntity]):
+    intents, entities = workflow.output
```

Comment:
lets call the 2nd arg as `entities` and these as `collected_entities`.

Reply:
updated it to:

```python
def update_entities(workflow: XLMRWorkflow, entities: List[BaseEntity]):
    intents, collected_entities = workflow.output
    workflow.output = (intents, collected_entities + entities)
```